articles/tutorials/web-apps-vs-web-apis-cookies-vs-tokens.md

```diff
@@ -12,6 +12,6 @@
 
 * For both approaches you can get the **same amount of information from the user**. That's controlled by the `scope` parameter sent in the login request (either using the [Auth0Lock](/lock), our [JavaScript library](https://github.com/auth0/auth0.js) or a plain link). The `scope` is a parameter of the `.signin({scope: 'openid name email'})` method which ends up being part of the querystring in the login request. You can get more details about this in the [Scopes Documentation](/scopes).
 
-* By default we use `scope=openid` in **token-based authentication** to avoid having a huge token (since the token will be traveling in the `Authorization` header for each request as mentioned above). You can control the attributes that you want to get in the token by doing `scope=openid name email another_attribute`.
+* By default we use `scope=openid` in **token-based authentication** to avoid having a huge token. You can control the attributes that you want to get in the token by doing `scope=openid name email another_attribute`.
 
 * Finally, you can **mix token-based authentication** with **cookie-based authentication**. Take into account that cookies will work just fine if the web app and the API are served from the same domain, so you might not need token based authentication. Now, if you need to, we also return a JWT on the web app flow. Each of our SDK will do it differently. If you want to call your APIs from JavaScript (instead of using the existing cookie) then somehow you have to set the `id_token` in your webpage. One way of doing it is by setting it on your layout/master page something like `window.token = ${"<%= id_token %>;"}` and then you get it from anywhere in your JavaScript code.
```

Comment:
@cabralmartin Can you have a look into the last bullet of this doc? I think it should be updated but I'm not sure if I should just replace `id_token` with `access_token`.